wiki/Ranking_Criteria/en.md

```diff
@@ -112,6 +112,7 @@ All forms of Artist/Title standardisation apply to both the `Romanised` and `Uni
 - **When a fictional character is credited as the singer of a song, the artist field is to be formatted in a `Character (CV: Voice Actor)` format.** For live action, credit the voice actor only.
 - **If the song is `TV size`, use a `(TV Size)` marker at the end of the current title string.** If there is an existing `TV size` marker in the title, the `(TV Size)` marker would replace it. Additionally, songs with existing `Short Ver` or `Game Ver` markers in their titles must use `(Short Ver.)` or `(Game Ver.)` markers.
   - Note: If a mapset's song contains matching sections to the `TV size`/`Short Ver`/`Game Ver` song, in the same order, and is roughly the same length as the official cut song, the mapset's edit will also count as a `TV Size`/`Short Ver`/`Game Ver` respectively. Covers and remixes do not count.
+- **If a song has been edited to have a higher tempo, use a `(Sped Up Ver.)` marker at the end of the current title string.** If there is an existing `Sped Up Ver` marker in the title, the `(Speed Up Ver.)` marker would replace it. Sped up songs in Techno, Trance, Hardstyle, or other similar genres must use a `(Nightcore Mix)` marker instead.
```

Comment:
last part (or just the "must" rly) doesnt seem decided in the thread

Reply:
also u misspelled the third instance of tag

Reply:
brought it up on thread